### PR TITLE
Tweak/center break content

### PR DESCRIPTION
**Pull request by mfyz**
### Requirements

- [ ]  issue was opened to discuss proposed changes before starting implementation.

These changes are no fix for issues, additions or minor tweaks.

- [x]  during development, `node` version specified in `package.json` was used (ie using [nvm](https://github.com/creationix/nvm)).
- [x]  package versions and package-lock.json were not changed (`npm install --no-save`).
- [x]  app version number was not changed.
- [x]  all new code has tests to ensure against regressions.
- [x] `npm run lint` reports no offenses.
- [x] `npm run test` is error-free.
- [x]  README and CHANGELOG were updated accordingly.

### Description of the Change

"full screen" (and fitscreen) breaks statically positon the break title, idea and progressbar in the screen. I've added a wrapper to the static content (excluding buttons) and css-centered vertically in the screen.

Added 4-5 missing Turkish translation strings that were in english in tr locale file.

### CODE REVIEW

**Comment by hovancik** [comment]
Not sure if I like it centered. I'll try running it for few days. 

image

**Comment by mfyz** [comment]
I have 2 screens with different orientations (one of them vertical long screen). seeing the break text fixed makes it only show on the top 20% of the screen. by default, when a full screen content comes with a lot of negative space, I focus on the center.

**Comment by mfyz** [comment]
I can also convert this as advanced setting flag.

**Comment by hovancik** [comment]
I think way forward is to have this as different style for fullscreen breaks? What do you think? I like current style for small window. You're right that fullscreen is not ideal.

File: app/break.html [hunk 1]
```diff
@@ -7,10 +7,12 @@
...
-      <div class="break-idea"></div>
-      <div class="break-text"></div>
-      <progress id='progress' max="10000" ></progress><br/>
-      <p id='progress-time'></p>
+			<div class="center-content">
+				<div class="break-idea"></div>
+				<div class="break-text"></div>
+				<progress id='progress' max="10000" ></progress><br/>
+				<p id='progress-time'></p>
+			</div>
```
Comment (pan93412) [inline app/break.html]: Please make sure that indent style is all unified.

File: app/css/app.css [hunk 2]
```diff
@@ -41,6 +41,13 @@ body.darwin {
   padding-top: 10%;
 }
 
+.microbreak .center-content {
+	position: absolute;
+	top: 50%;
+	transform: translateY(-50%);
+	width: 80%;
+}
```
Comment (pan93412) [inline app/css/app.css]: indent style.

use this regex to replace these: `s/\t/  /g`

File: app/locales/tr.json [hunk 3]
```diff
@@ -42,12 +42,12 @@
     "notificationStatus": "\nBeni Rahatsız Etme modunda"
   },
   "break": {
-    "postpone": "…wait for a bit",
```
Comment (pan93412) [inline app/locales/tr.json]: I suggest to separate this commit from this PR.

File: app/microbreak.html [hunk 4]
```diff
@@ -7,9 +7,11 @@
...
   <body>
     <div class="microbreak">
-      <div class="microbreak-idea"></div>
-      <progress id='progress' max="10000" ></progress><br/>
-      <p id='progress-time'></p>
+			<div class="center-content">
+				<div class="microbreak-idea"></div>
+				<progress id='progress' max="10000" ></progress><br/>
+				<p id='progress-time'></p>
+			</div>
```
Comment (pan93412) [inline app/microbreak.html]: indent

**Comment by hovancik** [comment]
closing as I just merged redesign

**Pull request closed without merging**